recipe/meta.yaml

```diff
@@ -2,37 +2,47 @@
 {% set version = "0.55.0" %}
 
 {% set name = goname.split('/')[-1] %}
-{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}  # src/github.com/junegunn/fzf
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
-  - folder: {{ pkg_src }}
-    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
-    sha256: 805383f71bca7f8fb271ecd716852aea88fd898d5027d58add9e43df6ea766da
+  folder: {{ pkg_src }}
+  url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+  sha256: 805383f71bca7f8fb271ecd716852aea88fd898d5027d58add9e43df6ea766da
 
 build:
   number: 0
+  skip: true  # [s390x]
   script:
     - pushd {{ pkg_src }}
-    - go build -ldflags "-X main.revision=conda-forge" -v -o $PREFIX/bin/fzf  # [not win]
-    - go build -ldflags "-X main.revision=conda-forge" -v -o %LIBRARY_BIN%\fzf.exe  # [win]
+    - go build -ldflags "-X main.revision=anaconda" -v -o $PREFIX/bin/fzf  # [not win]
+    - go build -ldflags "-X main.revision=anaconda" -v -o %LIBRARY_BIN%\fzf.exe  # [win]
     - bash -c "mkdir -p ${PREFIX}/share/fzf/shell; cp -v shell/* ${PREFIX}/share/fzf/shell/"
     - bash -c "mkdir -p ${PREFIX}/share/fzf/plugin; cp -v plugin/* ${PREFIX}/share/fzf/plugin/"
-    - go-licenses save . --save_path ../../../../library_licenses
+    - bash -c "go-licenses save . --save_path ${SRC_DIR}/library_licenses"
     # Clear out cache to avoid file not removable warnings
     - chmod -R u+w $(go env GOPATH) && rm -r $(go env GOPATH)  # [unix]
 
 requirements:
   build:
     - {{ compiler('go') }}
+    - posix  # [win]
```

Comment:
I def think it's better to use windows cmd syntax in the tests rather than bash syntax and pull in an extra dependency.

Reply:
`posix` pulls in all the msys2 packages it looks like. I haven't checked whether that would be a problem, but I think it's probably worth avoiding.